articles/aks/howto-deploy-java-liberty-app.md

```diff
@@ -40,9 +40,17 @@ Use the [az acr create](/cli/azure/acr?view=azure-cli-latest&preserve-view=true#
 az acr create --resource-group java-liberty-project --name youruniqueacrname --sku Basic --admin-enabled
```

Comment:
> Make sure *youruniqueacrname* is unique within Azure.

@gcharters  Yes, the acr name is scoped to the Azure. I verified that creating an acr with the same name using a different subscription failed.

Reply:
OK, thanks.  I think I also later discovered with later cos the ACR gets a public address, IIRC.